.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+  - repo: local
+    hooks:
+      - id: golines
+        name: golines
+        files: "go/.+$"
+        language: system
+        entry: |
+          sh -c 'cd ./go && go run github.com/segmentio/golines -w .'
```

Comment:
Is this taken from the monorepo? We don't have a `go` directory here.

Reply:
Ah, yes. I copied it from there.

Reply:
Fixed.